src/protocols/secure_channel/CASESession.cpp

```diff
@@ -235,7 +237,9 @@ CHIP_ERROR CASESession::EstablishSession(const Transport::PeerAddress peerAddres
     SetPeerAddress(peerAddress);
     SetPeerNodeId(peerNodeId);
 
+    PW_TRACE_START("CASESession::SendSigma1", "Commissioning");
```

Comment:
Why is this INSTANT not START/END ?

Why not PW_TRACE_SCOPE inside of SendSigmaR1?